sync-core/src/main/java/com/cloudant/sync/query/QuerySqlTranslator.java

```diff
@@ -468,6 +498,27 @@ protected static SqlParts whereSqlForAndClause(List<Object> clause, String index
         return SqlParts.partsForSql(where, parameterArray);
     }
 
+    private static String buildWhereClauseArgument(List<Object> values,
```

Comment:
Created an FB defect for placeholder limit logic to be handled at a later time:  https://cloudant.fogbugz.com/f/cases/45063/Add-placeholder-limit-logic-to-where-clause-generation